frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -1118,9 +1123,13 @@ function ReaderDictionary:downloadDictionary(dict, download_location, continue)
         return false
     end
 
-    local ok, error = Device:unpackArchive(download_location, self.data_dir)
+    -- stable target directory is needed so we can look through the folder later
+    local dict_path = self.data_dir .. '/' .. dict.name
```

Comment:
Very much a nit, but please stick to the double quotes style unless there's a good reason to deviate.